src/Integration.UnitTests/Persistence/SolutionBindingDataWriterTests.cs

```diff
@@ -58,19 +59,11 @@ public void TestInitialize()
         }
 
         [TestMethod]
-        public void Ctor_NullFileLoader_Exception()
+        public void MefCtor_CheckIsExported()
         {
-            Action act = () => new SolutionBindingDataWriter((ISolutionBindingFileLoader)null, null);
```

Comment:
MEF-imported components never have `null`s passed in the `ImportingConstructor` so I've removed the null-args  check and tests.